h5g_group_test.go

```diff
@@ -33,14 +32,12 @@ func TestGroup(t *testing.T) {
 		t.Errorf("wrong Name for group: want %q, got %q", "/foo", g1.Name())
 	}
 
-	err = f.CheckLink("/foo")
-	if err != nil {
+	if !f.LinkExists("/foo") {
 		t.Fatalf("err: %s", err)
```

Comment:
There is no error here.

Reply:
OOPS, sorry about this. Fixed it in the new commit